src/xrAICore/Navigation/level_graph_inline.h

```diff
@@ -591,8 +599,7 @@ IC void CLevelGraph::iterate_vertices(
     else
         E = end;
 
-    for (; I != E; ++I)
-        predicate(*I);
+    return {I, E};
```

Comment:
Codestyle: always leave whitespace between `{}` if they are non-empty. (and always leave no whitespace, if they are empty)
 ```suggestion
    return { I, E };
```